### PR TITLE
[TCSACR-163]Add floating keyboard API guide

### PR DESCRIPTION
**Pull request by Inhong**
### Change Description ###
Add New guide for floating keyboard

### API Changes ###
ACR: TCACR-163

### CODE REVIEW

File: docs/application/dotnet/guides/text-input/input-method.md [hunk 1]
```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
     }
     ```
 
+## Making a Floating Keyboard
```
Comment (sabeena21) [inline docs/application/dotnet/guides/text-input/input-method.md]: Suggestion: 
Can we change the heading as : Use Floating Keyboard
As per my understanding we don't make/create floating keyboard.

File: docs/application/dotnet/guides/text-input/input-method.md [hunk 2]
```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
...
+
+To make the floating keybaord to allow the user to move the keyboard:
```
Comment (sabeena21) [inline docs/application/dotnet/guides/text-input/input-method.md]: Please change to:
To use the floating keyboard to allow the user to move the keyboard:

Please can you verify whether the sentence is technically correct?

File: docs/application/dotnet/guides/text-input/input-method.md [hunk 3]
```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
...
+
+1.  Call the `SetFloatingMode()` method.
```
Comment (sabeena21) [inline docs/application/dotnet/guides/text-input/input-method.md]: Please replace period with colon.

File: docs/application/dotnet/guides/text-input/input-method.md [hunk 4]
```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
...
+
+    ```
+    public static void SetFloatingMode()
+    {
+        InputMethodEditor.SetFloatingMode(true);
+    ```
+
+2.  Create a container of PanGestureRecognizer to detect a dragging event.
```
Comment (sabeena21) [inline docs/application/dotnet/guides/text-input/input-method.md]: Please replace period with colon.

File: docs/application/dotnet/guides/text-input/input-method.md [hunk 5]
```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
...
+
+    ```
+    var panGesture = new PanGestureRecognizer ();
+    panGesture.PanUpdated += OnPanUpdated;
+    GestureRecognizers.Add (panGesture);
+    ```
+
+3.  Call the SetFloatingDragStart() and SetFloatingDragEnd() method when a drag event is received.
```
Comment (sabeena21) [inline docs/application/dotnet/guides/text-input/input-method.md]: Please replace period with colon.

Please add code tag for `SetFloatingDragStart()`  and `SetFloatingDragEnd()`

**Comment by Inhong** [comment]
Dear sabeena21.
Please check the fixed patch.

File: docs/application/dotnet/guides/text-input/input-method.md [hunk 6]
```diff
@@ -86,27 +86,27 @@ To start the IME application life-cycle:
     }
     ```
 
-## Making a Floating Keyboard
+## Use Floating Keyboard
 
-To make the floating keybaord to allow the user to move the keyboard:
+To use the floating keybaord to allow the user to move the keyboard:
```
Comment (sabeena21) [inline docs/application/dotnet/guides/text-input/input-method.md]: Please fix the typo error in the word "keyboard"

File: docs/application/dotnet/guides/text-input/input-method.md [hunk 7]
```diff
@@ -88,7 +88,7 @@ To start the IME application life-cycle:
 
 ## Use Floating Keyboard
```
Comment (sabeena21) [inline docs/application/dotnet/guides/text-input/input-method.md]: Please can you change "Use" to "Using" to maintain consistency with the entire document.

**Comment by Inhong** [comment]
Dear sabeena21.
Please check a new patch.

**Comment by mijin85cho** [comment]
Thanks for the review & update.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**